### PR TITLE
Add --preferred option for output

### PR DESCRIPTION
**Pull request by Jaakkonen**
Adds `--preferred` to `--output` arguments. Used to debug in #11 (which ended up being driver problem though). Adds a simple way to set a display to default preferred resolution.

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -486,6 +487,25 @@ static bool parse_output_arg(struct randr_head *head,
...
+		struct randr_mode *mode;
+		wl_list_for_each(mode, &head->modes, link) {
+			if (mode->preferred) {
+				found = true;
+				break;
+			}
+		}
+
+		if (!found) {
+			fprintf(stderr, "no preferred mode found");
```
Comment (emersion) [inline main.c]: Missing final `\n` here.

**Review by emersion — CHANGES_REQUESTED** [review]
LGTM apart from this nit!

File: main.c (hunk 1 above)
Comment (Jaakkonen) [reply main.c]: Updated commit. Now has the new line :)

**Review by emersion — APPROVED** [review]
Thanks!

**Pull request merged**